govc/test/events.bats

```diff
@@ -54,6 +54,21 @@ load test_helper
   run govc events vm
   assert_success
   [ ${#lines[@]} -ge $nevents ]
+
+  run govc events -type VmPoweredOffEvent -type VmPoweredOnEvent "vm/$vm"
+  [ ${#lines[@]} -eq 0 ]
+
+  run govc vm.power -on "$vm"
+  assert_success
+
+  run govc events -type VmPoweredOffEvent -type VmPoweredOnEvent "vm/$vm"
+  [ ${#lines[@]} -eq 1 ]
+
+  run govc vm.power -off "$vm"
+  assert_success
+
+  run govc events -type VmPoweredOffEvent -type VmPoweredOnEvent "vm/$vm"
+  [ ${#lines[@]} -eq 2 ]
```

Comment:
You could do with an additional VM operation (guest operation or reconfigure or something) so that you can assert that you do _not_ get an event for it.

Reply:
Several events such as VmCreatedEvent, VmMacAssignedEvent, etc, are being filtered out here.. but yes could make it more explicit within the test.